### ISSUE TITLE
SRM_PRINT_PARAMETERESTIMATES() omits duplicated RHS

### ISSUE THREAD

**Issue by TDJorgensen** [alexanderrobitzsch/srm#1]
Thanks for the great R package!  

I found an issue with the `summary()` method when printing covariances, which could also occur when there are cross-loadings or whenever the same predictor explains multiple outcomes.  On line [158](https://github.com/alexanderrobitzsch/srm/blob/master/R/SRM_PRINT_PARAMETERESTIMATES.R#L158) you check for duplicates, which inadvertently returns `TRUE` for duplicated `rhs` variables.  I expect this is easily fixed by replacing lines 158-160 with the following:

```
idx <- intersect(which(duplicated(M[,1])), lhs.idx)
if (length(idx)) M <- rbind(M[-idx, ])
```

I did not do a deeper dive to check whether this would prevent another possible error:  
If the `lhs` is duplicated, but a different `lhs` variable/component appears in the `$parm.table` between the original and duplicated element, then omitting those rows of `M` will lead to confusing output (i.e., the `rhs` element will appear to be matched with the wrong `lhs` element).  But it looks like regardless of the order parameters are specified in the syntax, you seem to sort them by their matrix indices so that all `lhs` should be clustered together, avoiding this potential problem.

**Comment by alexanderrobitzsch** [comment]
Thanks for notifying it. The bug will be fixed in the next version.

**Comment by TDJorgensen** [comment]
I just noticed the updated version.  I should apologize for not providing a reproducible example before, but there are still parameters not being printed in the `summary()` output.  I am unsure whether you already tried to address this issue, but here is an example now, estimating a saturated SRM for the 3 variables in the help-page data:

```
mod <- ' %Person

w1@A =~ 1*Wert1@A
w2@A =~ 1*Wert2@A
w3@A =~ 1*Wert3@A
Wert1@A ~~ 0*Wert1@A
Wert2@A ~~ 0*Wert2@A
Wert3@A ~~ 0*Wert3@A
w1@P =~ 1*Wert1@P
w2@P =~ 1*Wert2@P
w3@P =~ 1*Wert3@P
Wert1@P ~~ 0*Wert1@P
Wert2@P ~~ 0*Wert2@P
Wert3@P ~~ 0*Wert3@P
Wert1@A ~~ 0*Wert1@P
Wert2@A ~~ 0*Wert2@P
Wert3@A ~~ 0*Wert3@P
w1@A ~~ w2@A
w1@A ~~ w3@A
w2@A ~~ w3@A
w1@P ~~ w2@P
w1@P ~~ w3@P
w2@P ~~ w3@P
w1@A ~~ w2@P
w1@A ~~ w3@P
w2@A ~~ w3@P
w2@A ~~ w1@P
w3@A ~~ w1@P
w3@A ~~ w2@P

%Dyad

w1@AP =~ 1*Wert1@AP
w2@AP =~ 1*Wert2@AP
w3@AP =~ 1*Wert3@AP
Wert1@AP ~~ 0*Wert1@AP
Wert2@AP ~~ 0*Wert2@AP
Wert3@AP ~~ 0*Wert3@AP
w1@PA =~ 1*Wert1@PA
w2@PA =~ 1*Wert2@PA
w3@PA =~ 1*Wert3@PA
Wert1@PA ~~ 0*Wert1@PA
Wert2@PA ~~ 0*Wert2@PA
Wert3@PA ~~ 0*Wert3@PA
Wert1@AP ~~ 0*Wert1@PA
Wert2@AP ~~ 0*Wert2@PA
Wert3@AP ~~ 0*Wert3@PA
w1@AP ~~ intra1*w2@AP
w1@AP ~~ intra2*w3@AP
w2@AP ~~ intra3*w3@AP
w1@PA ~~ intra1*w2@PA
w1@PA ~~ intra2*w3@PA
w2@PA ~~ intra3*w3@PA
w1@AP ~~ inter1*w2@PA
w1@AP ~~ inter2*w3@PA
w2@AP ~~ inter3*w3@PA
w2@AP ~~ inter1*w1@PA
w3@AP ~~ inter2*w1@PA
w3@AP ~~ inter3*w2@PA
'
fit.satur <- srm::srm(mod, data = data.srm01,
                      rrgroup_name = "Group")
summary(fit.satur) # missing covariance estimates
```

Notice that only the first set of covariance estimates are printed at both `%Person` and `%Dyad` levels (i.e., first columns of `PHI_U` and `PHI_D`).  Some row names are printed for remaining columns, but without estimates or righthand-side variable/component names.

Thanks again for your work on this package!  I hope you are staying safe and healthy during this crisis.

**Comment by alexanderrobitzsch** [comment]
Thank you, Terrence. Yes, I agree. I worked on other issues and encountered some other issues. I will try to fix it in the CRAN version which hopefully should be submitted next week. 

I fixed the omission for regression coefficients, but apparently this is also needed for other operators.

Hopefully, there is a chance to see each other at some conference or meeting, probably not before next year.
